README

```diff
@@ -37,6 +37,15 @@ To package the browser extension for Firefox or Chrome:
 
 	$ make webwormhole-ext.zip
 
+To run the server using docker:
+	$ docker build -t webwormhole .
+	$ docker run -p 8000:80 webwormhole
+
+To use a custom signalling server.
+	$ ww -signal http://localhost:8000 send test.txt
+	east-pep-aloe
```

Comment:
I'm not sure what this means in this case.

Reply:
That's the wormhole phrase that's printed after you run send.